l10n_be_cooperator/__manifest__.py

```diff
@@ -35,4 +35,5 @@
     ],
     "installable": True,
     "application": False,
+    "auto-install": True,
```

Comment:
Is `cooperator_website` needed in the depends here ? If so, the localization will only be installed if all 4 modules are installed. 

Besides the auto_install feature, it also means we can't install the localization without installing cooperator_website.

Same story for other localizations.

Reply:
_En passant_, `l10n_be_invoice_bba` is empty in v14 so we can skip that dependency there.

Reply:
> Is `cooperator_website` needed in the depends here ? If so, the localization will only be installed if all 4 modules are installed.
> 
> Besides the auto_install feature, it also means we can't install the localization without installing cooperator_website.
> 
> Same story for other localizations.

Yes it is needed because the module overload the subscription form template

Reply:
> _En passant_, `l10n_be_invoice_bba` is empty in v14 so we can skip that dependency there.

Noted

Reply:
Ok then I think you should add in _known_caveats_ of `ROADMAP.rst` that this module will have to be split if someone needs to install one module without the other.

Reply:
Done